tests/ci/get_robot_token.py

```diff
@@ -84,13 +90,14 @@ def get_best_robot_token(tokens_path: str = "/github-tokens") -> str:
             )
             continue
         logging.info("Get token with %s remaining requests", rest)
-        if ROBOT_TOKEN is None:
-            ROBOT_TOKEN = Token(user, value, rest)
-            continue
-        if ROBOT_TOKEN.rest < rest:
-            ROBOT_TOKEN.user, ROBOT_TOKEN.value, ROBOT_TOKEN.rest = user, value, rest
-
-    assert ROBOT_TOKEN
+        if best_token is None:
+            best_token = Token(user, value, rest)
+        elif best_token.rest < rest:
+            best_token = Token(user, value, rest)
+        if best_token.rest > 300:
```

Comment:
This is a very brave assumption. Our `changelog` tasks consume requests like mad. As well as incremental sync.

Together with `shuffle`, it should still be good, though. `1000` is safe enough.